app/assets/stylesheets/application.css

```diff
@@ -1,19 +1,3 @@
-/*
- * This is a manifest file that'll be compiled into application.css, which will include all the files
- * listed below.
- *
- * Any CSS (and SCSS, if configured) file within this directory, lib/assets/stylesheets, or any plugin's
- * vendor/assets/stylesheets directory can be referenced here using a relative path.
- *
- * You're free to add application-wide styles to this file and they'll appear at the bottom of the
- * compiled file so the styles you add here take precedence over styles defined in any other CSS
- * files in this directory. Styles in this file should be added after the last require_* statement.
- * It is generally better to create a new file per style scope.
- *
- *= require_tree .
- *= require_self
- */
-
 .login_header {
   margin: 10px;
 }

```

Comment:
- [ ] Everything in your application seems to be okay. Devise is set up correctly as well. However, your application seems not to run when I start the server and view it on the browser. Below is the error that it is raising kindly look into it and fix it. Ensure that it is running correctly before you submit it for review.

![image](https://user-images.githubusercontent.com/86318284/216369279-d8288130-258a-4bf0-9d6b-1dd53afc3aa7.png)